src/app/pages/MediaAssetPage/MediaAssetPage.jsx

```diff
@@ -36,7 +39,9 @@ import { RequestContext } from '#contexts/RequestContext';
 import { GelPageGrid } from '#app/components/Grid';
 
 const StyledTimestamp = styled(Timestamp)`
-  padding-bottom: ${GEL_SPACING_DBL};
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
```

Comment:
This fix is the same one done for **PGL**s and **STYs** `StyledTimestamp` - they all restyle the `Timestamp` component in the same way. CodeClimate would flag it as repetition, but not sure it's worth tying these 3 components' styles to each other by pushing it into its own wee component.